packages/ag-charts-community/src/chart/interaction/interactionManager.ts

```diff
@@ -79,19 +73,19 @@ const CSS = `
 `;
 
 type SupportedEvent = MouseEvent | TouchEvent | Event;
-type ListenerMeta = { bypassPause?: PauseType[] };
+
+export enum InteractionState {
+    Default = -Infinity,
```

Comment:
nit: why do we need to set specific values on the enum?

Reply:
We don't really need to. The initial idea that I had in mind was that the number would represent priorities, hence why `Default = -Infinity` has the lowest priority and `Animation = 3` has the highest priority. But this was unnecessary in the end. I think that using a string union might actually make the code more readable.

Reply:
Actually powers of 2 is a good idea, because it would be much faster than iterating through an array 🤔

Reply:
Done. Changed number values to 1, 2, 4, 8